packages/cosmic-swingset/tools/test-kit.js

```diff
@@ -350,39 +282,73 @@ export const makeCosmicSwingsetTestKit = async (
     blockTxCount = 0;
     const context = { blockHeight, blockTime };
     await blockingSend({
-      type: ActionType.BEGIN_BLOCK,
+      type: SwingsetMessageType.BEGIN_BLOCK,
       ...context,
       params,
     });
-    await blockingSend({ type: ActionType.END_BLOCK, ...context });
-    await blockingSend({ type: ActionType.COMMIT_BLOCK, ...context });
-    await blockingSend({ type: ActionType.AFTER_COMMIT_BLOCK, ...context });
+    await blockingSend({ type: SwingsetMessageType.END_BLOCK, ...context });
+    await blockingSend({ type: SwingsetMessageType.COMMIT_BLOCK, ...context });
+    await blockingSend({
+      type: SwingsetMessageType.AFTER_COMMIT_BLOCK,
+      ...context,
+    });
     return getLastBlockInfo();
   };
+  await runNextBlock();
 
-  const makeQueueRecord = action => {
+  /** @type {InboundQueue} */
+  const actionQueue = makeQueue(actionQueueStorage);
+  /** @type {InboundQueue} */
+  const highPriorityQueue = makeQueue(highPriorityQueueStorage);
+  /**
+   * @param {{ type: QueuedActionType } & Record<string, unknown>} action
+   * @param {InboundQueue} [queue]
+   */
+  const pushQueueRecord = (action, queue = actionQueue) => {
     blockTxCount += 1;
-    return {
+    queue.push({
       action,
       context: {
         blockHeight: lastBlockHeight + 1,
         txHash: blockTxCount,
         msgIdx: '',
       },
+    });
+  };
+  /**
+   * @param {string | ((...args: any[]) => void)} fn
+   * @param {string} [jsonPermits] should deserialize into a BootstrapManifestPermit
+   * @param {InboundQueue} [queue]
+   */
+  const pushCoreEval = (fn, jsonPermits = 'true', queue = actionQueue) => {
```

Comment:
You likely want to default on the `highPriorityQueue`, that's what we do on the cosmos side.